setup.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+import locale
+import sys
+from os import getenv
+from subprocess import call
+
+# Start ignoring PyImportSortBear as imports below may yield syntax errors
+from coalib import assert_supported_version
+
+assert_supported_version()
+# Stop ignoring
+
+import setuptools.command.build_py
+from coalib.misc import Constants
+from coalib.misc.BuildManPage import BuildManPage
```

Comment:
Why ? dont need manpages.
Dont need Dbus

Also, you cannot use coalib.assert_supported_version here ! coala wouldnt be installed yet